packages/components/psammead-image/src/testHelpers/snapshotTests.jsx

```diff
@@ -58,6 +58,18 @@ const snapshotTests = (Component, additionalProps) => {
       width={landscape.width}
     />,
   );
+  if (Component.name === 'Img') {
```

Comment:
Why are you doing this `if`?

Reply:
This is to avoid creating extra test cases and snapshots for `AmpImg` without `width`. Since `width` is a required prop in `AmpImg`, my assumption is, it will always be provided. I therefore saw no benefit in the extra test cases and snapshots.